diff --git a/src/freedreno/vulkan/tu_device.cc b/src/freedreno/vulkan/tu_device.cc
index e2c0a64..84a1f60 100644
--- a/src/freedreno/vulkan/tu_device.cc
+++ b/src/freedreno/vulkan/tu_device.cc
@@ -222,7 +222,7 @@ get_device_extensions(const struct tu_physical_device *device,
       .KHR_variable_pointers = true,
       .KHR_vertex_attribute_divisor = true,
       .KHR_vulkan_memory_model = true,
-      .KHR_workgroup_memory_explicit_layout = true,
+      .KHR_workgroup_memory_explicit_layout = false,
       .KHR_zero_initialize_workgroup_memory = true,
 
       .EXT_4444_formats = true,
@@ -494,11 +494,11 @@ tu_get_features(struct tu_physical_device *pdevice,
    features->vertexAttributeInstanceRateDivisor = true;
    features->vertexAttributeInstanceRateZeroDivisor = true;
 
-   /* VK_KHR_workgroup_memory_explicit_layout */
-   features->workgroupMemoryExplicitLayout = true;
-   features->workgroupMemoryExplicitLayoutScalarBlockLayout = true;
-   features->workgroupMemoryExplicitLayout8BitAccess = true;
-   features->workgroupMemoryExplicitLayout16BitAccess = true;
+      /* VK_KHR_workgroup_memory_explicit_layout */
+   features->workgroupMemoryExplicitLayout = false;
+   features->workgroupMemoryExplicitLayoutScalarBlockLayout = false;
+   features->workgroupMemoryExplicitLayout8BitAccess = false;
+   features->workgroupMemoryExplicitLayout16BitAccess = false;
 
    /* VK_EXT_4444_formats */
    features->formatA4R4G4B4 = true;
